src/uct/ib/rc/base/rc_iface.h

```diff
@@ -76,6 +82,16 @@ struct uct_rc_iface {
         unsigned             tx_ops_mask;
         unsigned             rx_inline;
         uint16_t             tx_moderation;
+
+        /* Threshold to send "soft" FC credit request. The peer will try to
+         * piggy-back credits grant to the counter AM, if any. */
+        uint16_t             fc_sreq_thld;
+
+        /* Threshold to sent "hard" credits request. The peer will grant
+         * credits in a separate AM as soon as it handles this request. */
+        uint16_t             fc_hreq_thld;
```

Comment:
pls rename to "fc_hard_thresh" and "fc_soft_thresh" - because it all other places we use "thresh" abbreviation for "threshold" , and not "thld"